lib/blacklight/models/file.rb

```diff
@@ -11,11 +13,17 @@ class BlacklightFile
     def initialize(zip_entry)
       path = zip_entry.name
       id = File.basename(path)
+      xid = id[/__(xid-[0-9]+_[0-9]+)/, 1]
```

Comment:
I can't seem to find any documentation for matching strings this way. Do you know what it is called?